guides/source/security.md

```diff
@@ -1429,6 +1429,14 @@ Rails.application.config.content_security_policy do |policy|
 end
 ```
 
+Enable the [`report-to`][] directive to report violations to the specified Reporting API group:
+
+```ruby
+Rails.application.config.content_security_policy do |policy|
+  policy.report_to "csp-endpoint"
```

Comment:
For the guides specifically, can you provide an example that creates the `csp-endpoint` group?
Just so readers can tie the two together?